view/rooms/app.js

```diff
@@ -536,6 +538,8 @@ function displayBot(bot, botIndex) {
   // We don't want to scroll the iframe, just the bot inside it
   iframe.scrolling = 'no';
   iframe.frameBorder = 0;
+  iframe.sandbox = `allow-scripts allow-same-origin
```

Comment:
Cool! If there's no test for bot access parent, please either add one or create user story to add it.